modules/local/orthofinder_phylohogs.nf

```diff
@@ -0,0 +1,58 @@
+process ORTHOFINDER_PHYLOHOGS {
+    // label 'process_highthread'
+    label 'process_medium'
+    container "${ workflow.containerEngine == 'docker' ? 'austinhpatton123/orthofinder-2.5.4_r-4.2.2' : 
+        '' }"
+    stageInMode = "copy"
+    publishDir(
+        path: "${params.outdir}/orthofinder_phylohogs/",
+        mode: params.publish_dir_mode,
+        saveAs: { fn -> fn.substring(fn.lastIndexOf('/')+1) },
+    )
+    
+    input:
+    path species_tree        // Rooted species tree inferred using SpeciesRax
+    path orthogroups         // Orthofinder results directory inferred using MCL
+    path orthofinder_fastas  // Orthofinder-formatted fasta files
+    path orthofinder_seq_ids // Orthofinder sequence IDs
+    path orthofinder_spp_ids // Orthofinder species IDs
+    path speciesrax_gfts     // Reconciled gene family trees used in species tree inference
+    path generax_gfts        // Remaining reconciled gene family trees
+    path blast               // Blast similarity scores
```

Comment:
You need all of these?

Reply:
Sadly I do. Doesn't actively use them all, but it looks to make sure they're all there, and if not it has a fit and won't run.

Reply:
Smh OF